### PR TITLE
daemon: fix linter errors

### PR DESCRIPTION
**Pull request by mardy**

### CODE REVIEW

**Review by anonymouse64 — APPROVED** [review]
lgtm

File: daemon/api_model.go [hunk 1]
```diff
@@ -44,13 +44,6 @@ var (
 	}
 )
 
-type assertType int
-
-const (
-	serialType assertType = iota
```
Comment (bboozzoo) [inline daemon/api_model.go:50]: Looks like this was introduced back when `snap model` got added. @anonymouse64 do you recall if there was an idea to make use of those enums?

**Review by bboozzoo — APPROVED** [review]
LGTM

File: daemon/api_model.go (hunk 1 above)
Comment (anonymouse64) [reply daemon/api_model.go:50]: no idea, but if it's not needed anymore it was probably added as part of an initial implementation that got refactored away

**Pull request merged**